### DISCUSSION TITLE
Retrieving error codes for mssql when calling scan

### DISCUSSION THREAD

**Discussion by comsma** [uptrace/bun#781] [Q&A]
I'm using bun with with Microsoft sql and when my query returns an error from the database it returns it as a string without any error codes. I would like to use these error codes to return a status with grpc. Does bun handle these errors from the database or is this related to the go-mssqldb driver I am using?

**Comment by vmihailenco** [comment]
Bun does not do anything with the errors returned by the SQL driver. It must be something with go-mssqldb
